docs/root/start/install.rst

```diff
@@ -124,6 +124,17 @@ contains all contrib extensions on top of an Ubuntu base. The
 image contains all contrib extensions on top of an Ubuntu base as well as debug symbols. Throughout
 the documentation, extensions are clearly marked as being a contrib extension or a core extension.
 
+.. _install_tools:
+
+Tools images
```

Comment:
im wondering whether the contrib/tools images section would be better in the "using the docker image" or perhaps even better maybe we should create a separate page with the types of images and what is included

this is more a follow-up comment tho - i think lets land and iterate